packages/devextreme/testing/testcafe/tests/editors/overlays/toast.ts

```diff
@@ -32,6 +32,12 @@ const showToast = ClientFunction(
   },
 );
 
+const closeAllToasts = ClientFunction(() => {
```

Comment:
1) better use `hide` word instead of `close`
2) why shouldn't use our public [hideToasts()](https://js.devexpress.com/jQuery/Documentation/ApiReference/Common/Utils/ui/Methods/#hideToasts) method?

Reply:
fixed